README.md

```diff
@@ -214,9 +213,10 @@ Repo name to path mappings can be exact match (full name, full path) or wildcard
 
 An exact match for the full repo name to a full path takes priority over a matching wildcard, and wildcard matches must match to a wildcard path.
 
+An `:owner/:repo` template can be specified as a generic fallback.
+
 ```yaml
 repoPaths:
-  default_path: ~/code/repos # fallback value if none of the other paths match
   :owner/:repo: ~/src/github.com/:owner/:repo # template if you always clone github repos in a consistent location
   dlvhdr/*: ~/code/repos/dlvhdr/*       # will match dlvhdr/repo-name to ~/code/repos/dlvhdr/repo-name
```

Comment:
Are you suggesting removing the `owner/*` syntax? Or just updating this comment to
```suggestion
  dlvhdr/*: ~/code/repos/dlvhdr/*       # will match dlvhdr/:repo to ~/code/repos/dlvhdr/:repo
```
?

Reply:
Let's just remove it from the readme

Reply:
This is currently the only way to match all the repos under one specific owner. I'm hesitant to remove that from the README.

If you want to add support for `some-owner/:repo` as a key in `repoPaths` instead of `some-owner/*`, we can do that. But I'm a little more hesitant to introduce _that_ breaking change since unlike `default_path` (which was only added a few weeks ago), `some-owner/*` has been supported for almost a year (https://github.com/dlvhdr/gh-dash/commit/6ce3f89ab0d73dd88e359133699d1cf920f88699) and is potentially used more broadly.

Reply:
@dlvhdr I see you added a thumbs-up:

![image](https://github.com/dlvhdr/gh-dash/assets/1863540/8ece6bdc-1c21-4778-9a32-439e36aad5bd)

but I'm not sure how to interpret it 😅 

Could you let me know how you want to proceed? Thanks!

Reply:
I'm good to proceed without much suggestion :)

Reply:
Ok, sounds good! Thank you!